back_end/RayTracer/Models/Json/EnvironmentJSON.cs

```diff
@@ -11,11 +11,11 @@ namespace RayTracer.Models.Json
 {
     public class EnvironmentJSON
     {
-        string fileName;
-        WindowFrame winFrame;
-        ColorRGB background;
-        List<Light> lights;
-        Camera camera;
+        public string fileName;
```

Comment:
Looks like we're getting rid of the mutator/accessor methods throughout the files and making these variables public instead. I'm cool with that but just wanted to raise this to make sure it's been discussed with the others before we merge it in

Reply:
Checked in with Othman, this has been discussed and we're all cool with it.